README.md

```diff
@@ -302,6 +302,31 @@ const App = () => (
     <td colspan="3">The active dot smoothly slides to its new position, and the 'passive' dot smoothly fades away from the area it once occupied</td>
   </tr>
   <tr>
+   <tr>
+    <td colspan="4" align='center'>
+      <h3>Slides animations:</h3>
+    </td>
+  </tr>
+  <tr>
+    <td>slidesAnimation</td>
+    <td>Animate while transitioning between slides. This feature only works when a single slide is visible</td>
```

Comment:
- Animate while => The animation while
- We don't need the explanation that this feature only works when there is one slide as it's a general prop and there is also a `default` animation that includes more than one slide. Remove it